adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMediationAdapterVungle.h

```diff
@@ -18,16 +18,16 @@
 typedef NS_ENUM(NSInteger, GADMAdapterVungleErrorCode) {
   /// Missing server parameters.
   GADMAdapterVungleErrorInvalidServerParameters = 101,
-  /// An ad is already loaded for this network configuration. Vungle SDK cannot load
+  /// An ad is already loaded for this network configuration. Liftoff Monetize SDK cannot load
```

Comment:
It's still "Vungle SDK" in [Liftoff's documentation](https://support.vungle.com/hc/en-us/articles/360002925791), but I will defer to Liftoff team on whether it should be called Vungle or Liftoff Monetize SDK here.

Reply:
The same applies to the other places.

Reply:
@joshh-devrel In the [re-branding commit](https://github.com/googleads/googleads-mobile-ios-mediation/commit/1bee38f642f6b0cbbf9bbdd0ba66899be3d60a51), almost all comments were changed from `Vungle` to `Liftoff Monetize`. I thought that these cases were missed so I changed it. I'll remove the "SDK" part of the comment.